db/mysql/schema.sql

```diff
@@ -306,6 +355,12 @@ CREATE TABLE `usr_user_role`
   PRIMARY KEY (`user_id`, `role_id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+CREATE TABLE `usr_extuser` (
```

Comment:
original purpose was to decouple user id from sso provider from uv internal user id. this was requested by mrajniak.

On March 20, 2015 7:29:03 PM CET, Tomas Knap <notifications@github.com> wrote:
>> @@ -306,6 +355,12 @@ CREATE TABLE `usr_user_role`
>>    PRIMARY KEY (`user_id`, `role_id`)
>>  ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
>>  
>> +CREATE TABLE `usr_extuser` (
>
>What is the purpose of this table?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/UnifiedViews/Core/pull/282/files#r26864171

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.